src/dftd4/model.f90

```diff
@@ -113,20 +114,27 @@ module dftd4_model
    type(enum_ref), parameter :: d4_ref = enum_ref()
    !DEC$ ATTRIBUTES DLLEXPORT :: d4_ref
 
+   !> Create new dispersion model from molecular structure input
+   interface new_d4_model
+      module procedure :: new_d4_model_no_checks
+      module procedure :: new_d4_model_with_checks
+   end interface new_d4_model
+   !DEC$ ATTRIBUTES DLLEXPORT :: new_d4_model
```

Comment:
I think you have to export the two symbols individually, rather than the interface as the dllexport is specific to the routine the compiler dispatches to.

Reply:
Maybe @TyBalduf can confirm this?

Reply:
@marvinfriede it seems that dllexport can't be done on the interface as a whole and has to be done for each specific procedure. [Related Stackoverflow answer](https://stackoverflow.com/a/59629866/7696914)

Reply:
Thanks!